tempodb/encoding/block.go

```diff
@@ -81,7 +81,7 @@ func CopyBlock(ctx context.Context, meta *backend.BlockMeta, src backend.Reader,
 	}
 
 	// Bloom
-	for i := 0; i < common.GetShardNum(); i++ {
+	for i := 0; i < common.ValidateShardCount(int(meta.BloomShardCount)); i++ {
```

Comment:
this is weird b/c there are two different values here. what's in the meta and what's in common.ShardedBloomFilter. I think this method should return error if these values are different.

Reply:
Here the bloom is present only on the `src` backend in the byte representation. There's no `ShardedBloomFilter` representation, not sure if I'm missing something

Reply:
yup, you're right.